optimade_config.json

```diff
@@ -1,17 +1,11 @@
 {
     "debug": false,
-    "use_real_mongo": true,
-    "mongo_database": "optimade",
-    "mongo_uri": "mongodb+srv://MongoUser:MongoPassword@FakeMongoServer.com/optimade_database",
-    "links_collection": "optimade_links",
-    "references_collection": "optimade_references",
-    "structures_collection": "optimade_structures",
-    "default_db": "example_server",
-    "base_url": "https://optimade.example.org",
+    "default_db": "test_server",
+    "base_url": "http://localhost:5000",
```

Comment:
```suggestion
    "base_url": "http://localhost:5000",
    "log_dir": "/var/log/",
```

Reply:
I'm a little bit upset that we don't get our default values for our config anywhere in the docs, perhaps we could reinstate `default_config.json` until we have another solution?

Reply:
Fair enough.
I'll try and look into if we can retrieve the defaults from `SetupConfig` also. But as you stated in another PR, `pytkdocs` is not the best for this, unfortunately.

Reply:
I've done the "lazy" thing and just reinstated the `default_config.json` file in the root. Maybe I should move it to docs?
And then pasting it's contents into the new Configuration page at the very bottom over the list of configuration options. This being after a reference to `ServerConfig` in the API Reference ... :)

Reply:
I also updated the file to reflect the newest default values and configuration options from `ServerConfig`.

Reply:
I moved it to `docs/static/`.